docs/getting-started/gs-credentials.md

```diff
@@ -26,14 +26,15 @@ If you're using the integration environment as well as the production environmen
 
 ## API Key and Client Secret
 
-The API key and client secret allow the participant to connect to the [Operator Service](../ref-info/glossary-uid.md#gl-operator-service) and call API endpoints. These values identify the participant to the service.
+If you're using a server-side implementation, the API key and client secret allow you to connect to the [Operator Service](../ref-info/glossary-uid.md#gl-operator-service) and call API endpoints. These values identify you to the service.
```

Comment:
should we here link to the actual integration guides that belongs to server-side impl and likewise for client side impl mentioned below? just a bit concerned people may not know what client/server side impl means... also we can link the js express/standard guide accordingly until we restructure/rename them later

Reply:
@sunnywu: thx. For now I've added links in these two sections to each of these two guides. But for the future we'll update this to link to a page that clarifies client-side vs server-side and the integration options.